govcd/system.go

```diff
@@ -753,21 +753,14 @@ func (client *Client) GetStorageProfileByHref(url string) (*types.VdcStorageProf
 // 3. [NOT FOUND] The name matches, is not unique, but no Provider HREF was given: the search will fail
 // 4. [NOT FOUND] The name does not match any of the storage profiles
 func (vcdClient *VCDClient) QueryProviderVdcStorageProfileByName(name, providerVDCHref string) (*types.QueryResultProviderVdcStorageProfileRecordType, error) {
-	results, err := vcdClient.QueryWithNotEncodedParams(nil, map[string]string{
-		"type":     "providerVdcStorageProfile",
-		"pageSize": "128",
-	})
+
+	results, err := vcdClient.Client.cumulativeQuery(types.QtProviderVdcStorageProfile, map[string]string{
```

Comment:
Should we mention this improvement in the changelog? That now it supports more than 128 items.

Reply:
Thanks.
This is, in fact, a bug fix, and I mentioned it as such.